src/Types.php

```diff
@@ -221,10 +221,27 @@ public static function isFixedString($str)
         return strpos($str, 'fixedstring(') === 0;
     }
 
+    public static function isSimpleAggregateFunction($str)
+    {
+        return strpos($str, 'simpleaggregatefunction(') === 0;
+    }
+
+    protected function getTypeFromSimpleAggregateFunction($type)
+    {
+        if (self::isSimpleAggregateFunction($type)) {
+            preg_match('/\((.*?)\,\s?(.*)\)/', $type, $matches);
+            if (empty($matches)) {
+                return $type;
+            }
+            $type = trim(end($matches));
```

Comment:
> See if it can be changed into this? It will be more efficient
> 
> ```
> $type = trim(strstr($type, ','), ' ,)');
> ```

Yes, I'm will change it.